9c-main/chart/templates/remote-action-evaluator-headless.yaml

```diff
@@ -136,6 +136,8 @@ spec:
               fieldPath: metadata.namespace
         - name: JSON_LOG_PATH
           value: ./logs/$(POD_NAME)_$(NAMESPACE_NAME)_remote-action-evaluator-headless.json
+        - name: JSON_AEV_LOG_PATH
+          value: ./logs/$(POD_NAME)_$(NAMESPACE_NAME)_remote-action-evaluator-aev.json
```

Comment:
As a notice, I am not sure the environment affects to state services well. If it doesnt work it should pass it manually in StateService.cs